### ISSUE TITLE
DB Location is not Valid, Please Enter Correct Path

### ISSUE THREAD

**Issue by krishnasen2401** [linuxserver/docker-calibre-web#88]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

If you are new to Docker or this application our issue tracker is **ONLY** used for reporting bugs or requesting features. Please use [our discord server](https://discord.gg/YWrKVTn) for general support.

DB Location is not Valid, Please Enter Correct Path 
------------------------------

## Expected Behavior

Move forward to admin page after entering /books in location field
## Current Behavior

It says DB Location is not Valid, Please Enter Correct Path after entering /books
## Steps to Reproduce

1.deploy docker via portainer
2.open url(port8083) goes to /config page
3.enter /books
4. Says error in below title DB Location is not Valid, Please Enter Correct Path

## Environment
**OS:**      
**CPU architecture:** arm32/arm64      
**How docker service was installed:** via portainer      

## Docker logs

`[s6-init] making user provided files available at /var/run/s6/etc...exited 0.

[s6-init] ensuring user provided files have correct perms...exited 0.

[fix-attrs.d] applying ownership & permissions fixes...

[fix-attrs.d] done.

[cont-init.d] executing container initialization scripts...

[cont-init.d] 01-envfile: executing... 

[cont-init.d] 01-envfile: exited 0.

[cont-init.d] 10-adduser: executing... 

usermod: no changes

-------------------------------------

          _         ()

         | |  ___   _    __

         | | / __| | |  /  \ 

         | | \__ \ | | | () |

         |_| |___/ |_|  \__/

Brought to you by linuxserver.io

-------------------------------------

To support LSIO projects visit:

https://www.linuxserver.io/donate/

-------------------------------------

GID/UID

-------------------------------------

User uid:    911

User gid:    911

-------------------------------------
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 30-config: executing... 
cp: cannot stat '/app/calibre-web/gdrive.db': No such file or directory
[cont-init.d] 30-config: exited 0.
[cont-init.d] 99-custom-scripts: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.

I have tried puid and pgid to 1000 and 0 changed chmod of books and metadata.db

**Comment by CHBMB** [comment]
You aren't setting PUID/PGID properly. Still haven't filled out information we'd need.  This is for bugs, not support as the template says.  

Closing this use forums/discord for support.

**Comment by krishnasen2401** [comment]
Could you tell what is missing in the template cause the puid was set to 1000 in docker command -e puid 1000 so how was is it wrong I mentioned in the post that I have tried previous method who had same error and was solved why should i use discord when there is issues in GitHub seems like you want to keep it clean

**Comment by CHBMB** [comment]
Because we use Github for bugs, and this isn't a bug.

**Comment by mabushey** [comment]
@CHBMB Why are you so hostile and randomly claming crap?

**Comment by aptalca** [comment]
Because people don't read and don't provide the info we ask for specifically. We even set up a template form to fill out to make it as easy as possible. It gets very frustrating dealing with this every day. We all do this for fun but sometimes it becomes a chore.

**Comment by mabushey** [comment]
@krishnasen2401 This container is jacked up. The technosoft2000/calibre-web container worked on the first try for me.

**Comment by CHBMB** [comment]
@mabushey This container works fine.  Unless someone can prove otherwise with a proper bug report.  I use it myself daily.

If you can't post a bug report that we can look at, it's all just hot air and nonsense.

**Comment by cosmicbubble69** [comment]
I think it's a permissions issue but even when chowning the whole /books directory I couldn't get the basicconfig UI to accept /books so I moved by ebooks and metadata.db file to the /config directory and enter /config into the UI and it worked. basicconfig succeeded in finding the metadata.db in my /config directory and once I logged in all my books came up. By just moving the files in my /books directory to /config resulted in the DB Location is not Valid error going away.
